test/example/string_type/example_cont.f90

```diff
@@ -0,0 +1,9 @@
+program example_cont
```

Comment:
Maybe this file and (the specs counterpart) should be renamed to concatenation or just `example_concat`.